### ISSUE TITLE
Outdated collector config

### ISSUE THREAD

**Issue by Booster2ooo** [jufab/opentelemetry-angular-interceptor#7]
Hi,

The config provided in the example, at opentelemetry-angular-interceptor/blob/master/projects/example-app/collector/otel-collector-config.yaml, is outdated. Running it as such, with latest version of OTel Collector, will pop the following error:

```
otel-collector_1  | Error: cannot load configuration: error reading settings for receiver type "otlp": 1 error(s) decoding:
otel-collector_1  |
otel-collector_1  | * '' has invalid keys: endpoint
otel-collector_1  | 2020/07/17 07:28:28 application run finished with error: %v: cannot load configuration: error reading settings for receiver type "otlp": 1 error(s) decoding:
otel-collector_1  |
otel-collector_1  | * '' has invalid keys: endpoint
```

From what I understood, the following lines of the config:
```
receivers:
  otlp:
    endpoint: 0.0.0.0:9090
```

Should be replaced by something like:
```
receivers:
  otlp:
    protocols:
        grpc: 
            endpoint: 0.0.0.0:55680
        http: 
            endpoint: 0.0.0.0:55681 #9090 ?
```

**Comment by jufab** [comment]
Hi,

You right, new configuration in the latest version (0.6.0) must be :

```yaml
receivers:
  otlp:
    protocols:
        http: 
            endpoint: 0.0.0.0:9090 #default 55681 for http
```

I recommend that you do this for the moment : 

Change in docker-compose.yaml

```yaml
image: otel/opentelemetry-collector:latest
```
by

```yaml
image: otel/opentelemetry-collector:0.4.0
```

I try to fix with the latest release (0.6.0) but there is some issue with service.name...

**Comment by jufab** [comment]
I do a PR to fix the version.
now master is Ok
I complete an issue with the problem in the version 0.6.0 => https://github.com/open-telemetry/opentelemetry-collector/issues/1385
wait and see...
